content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/csi/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: Direct secrets
+description: >-
+  Mount secrets from Vault directly into application containers with the Vault
+  Secrets Operator CSI driver.
+---
+
+# Direct secrets
+
+@include 'alerts/enterprise-only.mdx'
+
+While the Vault Secrets Operator lets you store Vault secrets in clusters as native
+Kubernetes Secrets, you can also use the "direct secrets" feature to mount Vault secrets directly to containers as volumes.
```

Comment:
Can we explicitly call out that this new feature relies on the Kubernetes CSI and is implemented a CSI driver?

Reply:
I do so in the next line after this, including a link to the k8s CSI docs. After this first "transition" paragraph which I added to help connect the dots for HashiConf keynote viewers, the rest of the docs discuss it as a CSI driver.

Reply:
:+1: